common/logger/logger.ts

```diff
@@ -24,7 +25,22 @@ if (process.env.LOG_FORMAT === 'json') {
 
 configure({
     appenders: {
-        out: { type: 'stdout', layout: { type: 'coloured' } },
+        out: {
+            type: 'stdout',
+            layout: {
+                type: 'pattern',
+                pattern: '%[%x{transaction}%c%] %m%n',
```

Comment:
We can discuss which format we configure here, this is only used for local development anyways